torchgeo/datamodules/chesapeake.py

```diff
@@ -93,7 +93,7 @@ def __init__(
         """
         # This is a rough estimate of how large of a patch we will need to sample in
         # EPSG:3857 in order to guarantee a large enough patch in the local CRS.
-        self.original_patch_size = patch_size * 2
+        self.original_patch_size = patch_size * 3
```

Comment:
Just a heads up @calebrob6. Not sure what changed since the last release that made this necessary for the test to pass. We should really convert this to a normal RasterDataset.